samples/interactive-tutorials/prediction/get_prediction_with_filtering.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START prediction_for_products_with_filter]
+# Call Retail API to get predictions from Recommendation AI using filtering.
+#
+import os
+
+import google.auth
+from google.cloud.retail_v2 import (
+    PredictionServiceClient, UserEvent, ProductDetail,
+    Product, PredictRequest
+)
+from google.api_core.client_options import ClientOptions
+
+
+project_id = google.auth.default()[1]
+placement_id = os.environ["GOOGLE_CLOUD_PLACEMENT"]
```

Comment:
Please add noxfile.py. Also, populate this environment variable in `noxfile_config.py`.

Reply:
Wouldn't it also make sense to write this as a commented-out line and add a `TODO(Developer` comment here for the env variables in the samples? That way isn't hardcoded in `noxfile_config.py`, which I've noticed is the pattern (not putting it in `noxfile_config.py`. I'm wondering if it might impact testing if we have it hard-coded, also.

Reply:
@loferris In case of removing hard-coded `GOOGLE_CLOUD_PLACEMENT` from `noxfile_config.py` we should handle then getting this variable from `noxfile.py` like we do with `GOOGLE_CLOUD_PROJECT` env. I think it's redundant and better to leave as it is for tests when we have separately hard-coded `GOOGLE_CLOUD_PLACEMENT `. What about `placement_id` in samples - we could add a comment above this variable and give a possibility to user set it manually or via environment.

Reply:
Done

Reply:
@parthea Done